recipes/pymeep/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "pymeep" %}
+{% set version = "1.5" %}
+{% set sha256 = "2daedb5258365a9d492e08917bb99ce590dba63f86997cd8e5225b4a58425922" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: meep-{{ version }}.tar.gz
+  url: https://github.com/stevengj/meep/releases/download/v{{version}}.0/meep-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - gcc
```

Comment:
Please use the new compilers to to try to improve your build times: https://conda-forge.org/docs/meta.html?#compilers

Reply:
Please use the new syntax. It seems that `libctlgeom` is missing from this build, which is causing the errors.

Reply:
It finds `libctlgeom` with the old compilers. With the new syntax, the `PATH` doesn't seem to be set up correctly. `configure` can't find any dependencies (hdf5, gsl, zlib, harminv, etc.).
```
checking for harminv_get_freq_error in -lharminv... no
configure: WARNING: harminv support is disabled
checking mpb.h usability... no
checking mpb.h presence... no
checking for mpb.h... no
checking for cos in -lm... yes
checking for cblas_dgemm in -lgslcblas... no
checking for gsl_sf_bessel_Jn in -lgsl... no
configure: WARNING: Missing GNU GSL library...Bessel-function field initialization will not be supported.
checking for deflate in -lz... no
configure: WARNING: zlib is required for HDF5!
checking for H5Pcreate in -lhdf5... no
configure: WARNING: Couldn't find the HDF5 library!!  Switching to --without-hdf5.
```

Reply:
Also, the instructions @scopatz linked to above say each dependency should be built with the new compilers. This package has a lot of dependencies that don't seem to use the new compilers (e.g., [hdf5](https://github.com/conda-forge/hdf5-feedstock/blob/master/recipe/meta.yaml).

Reply:
Conda-Forge has not moved over to the new compilers, yet. The new syntax is just a wrapper around the current `toolchain` setup. However, when we do make the migration over to the new compilers we need the new syntax.

I think to fix the problem with that is causing the trace above is from not having a `host` section.
see: https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#host

Reply:
Using the new compilers did not improve the build times. It still times out on MacOS. Do I have to skip tests to ensure that the jobs complete in under 1h 20m, or can you recommend an alternative?